### ISSUE TITLE
Bus stops missing in POIs

### ISSUE THREAD

**Issue by eikes** [protomaps/basemaps#163]
The tileset does not include bus stops. It would be nice to have them.

**Comment by bdon** [comment]
Thanks for the PR! @nvkelso it looks like `bus_stop` is [defined in Tilezen](https://tilezen.readthedocs.io/en/latest/layers/#points-of-interest) for z16, which we don't do for tileset v3. So it makes sense for v3 to add this additively.

In general we should continue with additions that align with Tilezen, this one looks like it should have a minimal impact on overall tile size.

**Comment by nvkelso** [comment]
Looking at the Tilezen source code (ultimate source of truth versus docs), zoom 18 is more appropriate. Otherwise they really crowd the map!

- https://github.com/tilezen/vector-datasource/blob/master/yaml/pois.yaml#L1974-L1993

**Comment by bdon** [comment]
Need to add a visualization for bus stops, blocked on #160

**Comment by bdon** [comment]
@eikes this is in the latest daily build:

https://protomaps.github.io/PMTiles/?url=https%3A%2F%2Fbuild.protomaps.com%2F20231111.pmtiles#map=18.33/37.765429/-122.419588
<img width="1024" alt="Screenshot 2023-11-12 at 11 56 31" src="https://github.com/protomaps/basemaps/assets/77501/19572df1-4057-48e6-ad9c-ae118f72ab5d">

**Comment by eikes** [comment]
Nice!

**Comment by bdon** [comment]
Bus stop icons are in styles 4.2:
![CleanShot 2024-10-23 at 15 08 32@2x](https://github.com/user-attachments/assets/255389c4-dc10-4692-a51a-ddc5c4085559)
